<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paginaConferenza</class>
 <widget class="QWidget" name="paginaConferenza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>718</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Home">
    <widget class="QLineEdit" name="Nome">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelNome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAggiungi">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>690</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Aggiungi</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLuogo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Luogo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAcronimo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Acronimo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumeroPartecipanti">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero partecipanti</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="Data">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>450</y>
       <width>241</width>
       <height>155</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Luogo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="NumeroPartecipanti">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>400</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50000</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelOrganizzatori">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Organizzatori</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Acronimo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listConferenze">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>310</width>
       <height>420</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Organizzatori">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>211</width>
       <height>230</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelConferenzeInserite">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Conferenze inserite</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="buttonVisualizzaGuadagnoAnnualeConferenza">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>470</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualizza il guadagno annuale di una conferenza</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageVisualizzaGuadagnoAnnualeConferenza">
    <widget class="QCalendarWidget" name="page2_Data">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>241</width>
       <height>155</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="page2_labelData">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="page2_labelLuogo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Luogo</string>
     </property>
    </widget>
    <widget class="QListWidget" name="page2_listArticoli">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>279</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="page2_buttonCerca">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>420</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Cerca</string>
     </property>
    </widget>
    <widget class="QPushButton" name="page2_buttonIndietro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Indietro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="page2_NomeConferenza">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="page2_labelConferenza">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Conferenza</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="page2_Luogo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="page2_labelGuadagnoTotale">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>510</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Guadagno Totale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="page2_GuadagnoTotale">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>510</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
